kratos/testing/tester.cpp

```diff
@@ -228,8 +228,7 @@ namespace Kratos
 		{
 			// creating the regex pattern replacing * with ".*"
 
-#if defined(__GNUC__) &&( __GNUC__ < 4 || \
-              (__GNUC__ == 4 && (__GNUC_MINOR__ < 9))) 
+#if __cplusplus <= 199711L
```

Comment:
There was a problem in VS 2012 with this but since we already use 2015 it should be fine. I will test it again to be sure.